### PR TITLE
Documentation update (Readme.md , Code-of-conduct.md , Contributing.md Added)

### LINKED ISSUE #15: Documentation Update
- [ ] Basic of Fork,Clone & hitting a PR
- [ ] Setting up the Project Locally
- [ ] Code of Conduct
- [ ] Contributing Rules/Style (same as #13)

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
**Issue Number**

fixes #15 , #13, #23 

Guidede by : @Siddhant-K-code 

Edited the Readme.md file with all the required steps and specifications, Added Code-of-conduct.md, Added Contributing.md

React-scripts error for version solved, added react script for npm and yarn users so that they do not face ESOCKETTIMEOUT error. 

**Test plan**

All the links embedded are working and tested.

**Checklist**

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md
Comment (Abhishek-kumar09) [inline README.md:1]: I am really liking your readme, and all the emojis and everything, but it is too verbose.

You can create a new PR once PR https://github.com/codeforcauseorg/edu-client/pull/23 is merged to put your changes on top of it. 

For now save it somewhere else, and remove this change.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Revert all the changes in package.json, package-lock.json, and yarn.lock. Since it is under the scope of this PR.

File: README.md
Comment (kaiwalyakoparkar) [reply README.md:1]: @Siddhant-K-code Told me the structure according to that I created the readme, What should I do?

File: CODE_OF_CONDUCT.md
Comment (kaiwalyakoparkar) [reply CODE_OF_CONDUCT.md]: Done..

File: README.md
Comment (Abhishek-kumar09) [reply README.md:1]: No it's all fine, the readme is really looking beautiful, I am telling that please remove the changes of readme from this PR since https://github.com/codeforcauseorg/edu-client/pull/23 is already about to merge, so once that will be merged you can put your further changes after that.

For now remove all the readme changes from this PR.

Comment (kaiwalyakoparkar) [reply README.md:1]: Because according to him content in #23 pr is not prioritized corretly.

Comment (kaiwalyakoparkar) [reply README.md:1]: ok

Comment (kaiwalyakoparkar) [reply README.md:1]: Then I should only keep contributing.md for now?

Comment (Abhishek-kumar09) [reply README.md:1]: Yes

Comment (Siddhant-K-code) [reply README.md:1]: @Abhishek-kumar09 created PR with a lot of details, also he also informed first, #23 was with no assignment and also #23 has less info.

Comment (kaiwalyakoparkar) [reply README.md:1]: Yes as said @Abhishek-kumar09  I have created a pull request with only contributing.md. Please let me know what to do.

Comment (kaiwalyakoparkar) [reply README.md:1]: @Siddhant-K-code , @Abhishek-kumar09  Also I have documented the changes made in package.json, package-lock.json, and yarn.lock so that the user will not encounter issues further.

**Comment by kaiwalyakoparkar** [comment]
Please review the latest changes.

File: README.md
Comment (Siddhant-K-code) [inline README.md:67]: Line 67 & 68 should be like this :
image

File: README.md
Comment (Siddhant-K-code) [inline README.md:162]: I am not the official Maintainer as of now, hope @KeenWarrior will make me soon, till then remove my name from there

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]

File: README.md
Comment (kaiwalyakoparkar) [reply README.md:162]: ok

File: README.md
Comment (kaiwalyakoparkar) [reply README.md:67]: what should I do to make it look like that?

File: README.md
Comment (kaiwalyakoparkar) [reply README.md]: done

File: README.md
Comment (kaiwalyakoparkar) [reply README.md]: oh yes, done

**Comment by kaiwalyakoparkar** [comment]
Please review the latest changes.

**Comment by Abhishek-kumar09** [comment]
Thanks @kaiwalyakoparkar for the Pull request, thanks @Siddhant-K-code for the detailed review.
Looks perfect.

**Comment by Abhishek-kumar09** [comment]
@kaiwalyakoparkar Please rename the doc_images folder, you made the typo there

**Comment by Abhishek-kumar09** [comment]
Instead:
1. create a folder doc
2. make a subfolder images
and place your images inside that

**Comment by kaiwalyakoparkar** [comment]
> Instead:
> 
> 1. create a folder doc
> 2. make a subfolder images
>    and place your images inside that

ok

**Comment by Abhishek-kumar09** [comment]
Thank you!

File: README.md
Comment (Siddhant-K-code) [inline README.md:11]: Change `LICENSE.txt` to `LICENSE`

File: README.md
Comment (Siddhant-K-code) [inline README.md:69]: Change this from:

```
upstream        https://github.com/codeforcause/edu-client.git (fetch)
upstream        https://github.com/codeforcause/edu-client.git (push)
```

To: 

```
upstream        https://github.com/codeforcauseorg/edu-client.git (fetch)
upstream        https://github.com/codeforcauseorg/edu-client.git (push)
```

File: README.md
Comment (kaiwalyakoparkar) [reply README.md:11]: Resolved By @kunal-kushwaha in merged PR #49

File: README.md
Comment (Abhishek-kumar09) [reply README.md:69]: Thanks @Siddhant-K-code

**Pull request merged**
